### ISSUE TITLE
Better documentation of, &/or support for, other platforms

### ISSUE THREAD

**Issue by mjwestgate** [AtlasOfLivingAustralia/galah#126]
As of v1.4.0, `galah` supports the ALA (default), and 5 other atlases (Austria, Guatemala, Spain, Sweden & UK). Although other atlases exist (see the [full list](https://living-atlases.gbif.org)), it is unclear whether these other atlases are unsupported because of technical issues, or because we simply haven't added them yet. 

In increasing order of difficulty, a useful set of activities to address this might include:

1. [SIMPLE] List all living atlases within `show_all_atlases()`, and record whether they can be called by `galah`
2. Allow other atlases to be queried by data-retrieving functions such as `atlas_counts` and `atlas_occurrences`
3. Support queries to taxonomic name services of non-ALA atlases via `search_taxa` or `search_identifiers`, rather than requiring the user to search using `taxize`. This could work either by importing `taxize`, or by writing new code
4. [DIFFICULT] Support calls to the [GBIF API](https://www.gbif.org/developer/summary) using `galah` functions. This would require `galah_filter` to support queries other than `solr`. We would also need to either call `rgbif::occ` within `atlas_occurrences` to download the data, or write new code to achieve the same goal.

**Comment by mjwestgate** [comment]
As of commit a0e0b2e504b6e21271018a555aee85a2ddb974c4, we now support name matching for all atlases that use GBIF taxonomy (currently Austria, Guatemala, Spain & Sweden). This effectively means we can remove the advise to use `taxize` for this purpose from our documentation. What we can't do yet is query other GBIF APIs from `galah`, which leaves open the question of what to do next. Basically we have two options:

- Only support the living atlas community, by supporting name matching services from GBIF, but no other GBIF data/services
- Expand `galah` to retrieve record counts or occurrences from GBIF

Based on my reading of the documentation [here](https://www.gbif.org/developer/occurrence), there are a few points to consider in reaching a decision. For occurrence data:

- There is no API for field values, meaning that `show_all_values`/`search_values` won't work unless we hard-code some entries. This is effectively the approach taken by e.g. `rgbif::isocodes`
- GBIF don't allow the user restrict the columns returned in an occurrence download, meaning that `galah_select` would be either be ignored or applied post-download (which is inefficient)
- The occurrence download process for GBIF is fairly involved, but does support filtering on a large number of fields. The syntax is different from solr, in that it requires a [JSON-type structure](https://www.gbif.org/developer/occurrence#predicates) which we don't currently produce; but it could be made to work
- Downloads [appear limited to 101k records](https://www.gbif.org/developer/occurrence#download_limits), much less than the 50 million allowed by ALA. This is somewhat irrelevant as clearly people still want information from GBIF; but would require we update error messages etc.

For counts the same issues apply, plus a few more:
- The GBIF counts API only supports a narrow range of facets (effectively only `year` and `basisOfRecord`), which limits usefulness of `galah_group_by`, unless we write our own iteration code
- There is a hack to apply occurrence filters on counts, by using the `occurrence/search` API and adding `&limit=0`. This would maintain equivalence between `atlas_counts` and `atlas_occurrences`, which is an important feature.

In summary, while there is merit in supporting count and occurrence queries from GBIF; but the architecture is sufficiently different to make this somewhat challenging.

**Comment by mjwestgate** [comment]
There is a new Living Atlases website that lists current participants and their status [here](https://living-atlases.gbif.org/participants/). GBIF 'noted portals' are available [here](https://dev.gbif.org/hosted-portals.html), but it is unclear if there are other sites that use GBIF infrastructure hosted elsewhere.

**Comment by mjwestgate** [comment]
This is closed, with the caveat that different atlases perform differently, and GBIF not currently supported
